frontend/app/components/common/path-heleper/path-helper.service.js

```diff
@@ -186,6 +186,14 @@ function PathHelper() {
     apiV3AvailableProjectsPath: function() {
       return PathHelper.apiV3WorkPackagesPath() + '/available_projects';
     },
+    apiv3MentionablePrincipalsPath: function(projectId, term) {
+      var path = PathHelper.apiV3;
+      path = path + '/principals?filters=[{"status":{"operator":"!","values":["0","3"]}},{"member":{"operator":"=","values":["' + projectId + '"]}},{"type":{"operator":"=","values":["User"]}}';
```

Comment:
Please use the `ApiV3FilterBuilder` filter builder here. 

```
const filter = new ApiV3FilterBuilder()
filter.add('status', '!', values: '...');
filter.add(...)
```

This is only possible once the PathHelper is a typescript class. This will happen once https://github.com/opf/openproject/pull/5901 is merged